src/Fixer/PhpUnit/PhpUnitTestClassRequiresCoversFixer.php

```diff
@@ -151,5 +148,21 @@ protected function applyFix(\SplFileInfo $file, Tokens $tokens)
 
             $tokens[$docIndex] = new Token([T_DOC_COMMENT, implode('', $lines)]);
         }
+
+        $lines = $doc->getLines();
```

Comment:
this, till end of the method, is duplicated code. why is that? maybe a merge conflict solved wrongly?

Reply:
You're correct, seems like a rebase that went wrong somewhere, i fixed it now.